src/hooks/usePropsGetters/useSelectInputProps.ts

```diff
@@ -15,18 +15,18 @@ export default function useSelectInputProps(
   quizAnswerChanged: QuizEventsReturn.QuizAnswerChanged,
   nextQuestion: QuizEventsReturn.NextQuestion,
   currentQuestionData?: Question,
-  answerInputs?: AnswerInputState,
-  answers?: Answers
+  answerInputs?: AnswerInputState
 ): GetSelectInputProps {
   const type: `${QuestionTypes}` | undefined = currentQuestionData?.type;
   const hasImages = currentQuestionData?.options?.some((option: QuestionOption) => option.images);
 
   const [selected, setSelected] = useState<Selected>({});
-  const singleSelectQuestionAnswer = useRef({});
+  const singleSelectClicked = useRef({});
```

Comment:
Just out of curiosity, what's the benefit of `useRef` instead of `useState` here?

Reply:
Will leave it as useRef instead of useState since we don't need this to cause rerenders and only we want to keep track of whether singleSelectClicked or not